libsolidity/inlineasm/AsmParser.cpp

```diff
@@ -338,7 +339,7 @@ Parser::ElementaryOperation Parser::parseElementaryOperation(bool _onlySinglePus
 		else
 			literal = currentLiteral();
 		// first search the set of instructions.
-		if (!m_julia && instructions().count(literal))
+		if (m_flavour != AsmFlavour::IULIA && instructions().count(literal))
```

Comment:
Shouldnt strict mode disallow non-functional instructions?

Reply:
I'm pretty sure it does. I can check later, but there has to be a function that parses the `f`-part in `f(a, b)`, and I think it is this one here.

Reply:
Yep, double checked, the non-loose case is handled on line 191.

Reply:
So this is solved?

Reply:
This one is.